src/lib/dremel/tests/Flow/Dremel/Tests/Unit/DremelShredTest.php

```diff
@@ -103,25 +121,4 @@ public function test_dremel_shred_on_repeated_columns() : void
             ]
         );
     }
-
-    public function test_dremel_with_rows_with_combined_scalar_values_and_array_values() : void
```

Comment:
This constraint was incorrect since there is a scenario where arrays also contain nulls, not only scalar values.